### ISSUE TITLE
refactor(License Browser): remove "old" `license_list_files`-view

### ISSUE THREAD

**Issue by maximilianhuber** [fossology/fossology#598]
I would suggest, to remove the `license_list_files`-view, since the `license`-page covers the same functionality (up to some filtering) with the **flat**-view.

### Screenshots
The `license`-page in **flat**-view mode looks like
![screenshot](https://cloud.githubusercontent.com/assets/1187050/11898304/3078d33c-a597-11e5-81c5-800ac78f1f36.png)
and the corresponding `license_list_files`-view is the following
![old](https://cloud.githubusercontent.com/assets/1187050/11898142/f9a726d4-a595-11e5-91fb-d7588db7a7fc.png)

### Misc information:
The "old" view can be accessed by clicking on the numbers in the license histogram (marked in the following screenshot):
![view](https://cloud.githubusercontent.com/assets/1187050/11898253/cb01ee8a-a596-11e5-807a-e6976cd72168.png)
The "new" view is obtained by entering the **flat**-mode and aplying a filter.

**Comment by bufferoverflow** [comment]
:+1:

**Comment by hhts** [comment]
I would disagree that it provides the same functionality, at least if I understood correctly and you state that the flat view is in essence the same as clicking on scanner count numbers.

The scanner count numbers has a neat "show all" feature, which is visible top right in your screen shot also. It allows me to get a huge listing with all of the files under a certain license, then filter that list by subtracting filetypes and/or license categories. The flat view is currently capped at 100 matches per page and does not support exclusions.

**Comment by shaheemazmalmmd** [comment]
A discussion in community call needed to address this. to eliminate This page or if useful can be improved further.
